modules/meshrenderinggl/depends.cmake

```diff
@@ -0,0 +1,8 @@
+#--------------------------------------------------------------------
+# Dependencies for current module
+# List modules on the format "Inviwo<ModuleName>Module"
+set(dependencies
+    InviwoOpenGLModule
+    InviwoBaseModule
+    InviwoBaseGLModule
+)
```

Comment:
IMO, should add `set(EnableByDefault ON)` to this module

Reply:
Since it requires opengl 4.2 im not so sure